src/towncrier/_builder.py

```diff
@@ -106,10 +107,22 @@ def __call__(self, section_directory: str = "") -> str:
 def find_fragments(
     base_directory: str,
     config: Config,
+    strict: bool | None,
```

Comment:
I hope that we can remove this argument and just use the configuration value to control the behaviour of this method.

Reply:
I don't think so, because we have different behaviour for ``towncrier check`` vs ``towncrier build``. Unless you want to align their behaviours?